src/components/ShotForm/ShotForm.js

```diff
@@ -8,54 +8,86 @@ import FormError from "../FormError";
 import http from "../../utils/http";
 import Input from "../Input";
 import Layout from "../Layout";
+import Loader from "../Loader";
+import Select from "../Select";
 
 const ShotForm = props => {
-  const { errors, handleChange, handleSubmit, values } = useFormik({
+  const [isLoading, setIsLoading] = useState(false);
+
+  const options = [
```

Comment:
Nit: these can be taken out of the `ShotForm` component and into a constant that can be reused throughout the app?